src/parser/inlines.rs

```diff
@@ -837,13 +845,6 @@ impl<'a, 'r, 'o, 'd, 'i, 'c, 'subj> Subject<'a, 'r, 'o, 'd, 'i, 'c, 'subj> {
         opener_num_chars -= use_delims;
         closer_num_chars -= use_delims;
 
-        if self.options.extension.strikethrough
-            && opener_char == b'~'
-            && (opener_num_chars != closer_num_chars || opener_num_chars > 0)
```

Comment:
I suspect (though haven't checked) this will cause spec failure with [this example](https://github.github.com/gfm/#example-493). The spec requires three or more tildes to not generate a strikethrough; since we've already subtracted `use_delims` (2 for a strikethrough of length 2 or more) from `opener_num_chars`, if we had 3 or more, `opener_num_chars > 0` would be true and so we'd `return None` here.

Reply:
If subscript is enabled, it will be a subscript _as well as_ a strikethrough (if both enabled) like in cases of italics and bolds. Considering this I removed the condition.

Reply:
Adding and additional `&&` to check if subscript is enabled. Like
```rust
if (self.options.extension.strikethrough
    && opener_char == b'~'
    && (opener_num_chars != closer_num_chars || opener_num_chars > 0))
    && !self.options.extension.subscript
{
    return None;
}
```